cmd/swctl/app/cmd_manage.go

```diff
@@ -370,11 +372,32 @@ func runManageCmd(cli Cli, opts ManageOptions, args []string) error {
 		finalConf = mainConf
 	}
 
-	// format final output
 	if opts.Format == "" {
 		opts.Format = "yaml"
 	}
-	if err := formatAsTemplate(cli.Out(), opts.Format, finalConf); err != nil {
+
+	var fileName string
+	var writer io.Writer
+
+	if strings.HasSuffix(opts.File, opts.Format) {
+		fileName = opts.File
+	} else {
+		fileName = fmt.Sprintf("%s.%s", entityName, opts.Format)
+		if opts.File != "" {
+			logrus.Warnf("Defined output file name is not in required format. Naming %s will be used.", fileName)
```

Comment:
Situation when user of swctl forgets to set the format and give wrong output file i.e. `config.json` can happen. It was not in no way stated in command help and he doesn't find the file in directory due to completely changed name. This is quite confusing. We can make the problem that happened more visible to user by few steps:
1. update flag help by stating that the format from format flag will be enforced as suffix
2. we don't dismiss the naming completely, just add the correct suffix, i.e. `config.json` + yaml formatting -> `config.yaml` (or less desirable `config.json.yaml`)
3. change the log message to state that given format and filename suffix doesn't match and that other filename will be used.

PS: the default `<entity name>.<format flag suffix>` when no file flag is ok